modules/tide_jira/src/TideJiraAPI.php

```diff
@@ -275,29 +342,25 @@ private function getAuthorInfo(NodeInterface $node) {
    * @return string
    *   Templated ticket body as a Heredoc.
    */
-  private function templateDescription($name, $email, $department, $title, $id, $moderation_state, $bundle, $is_new, $updated_date, $notes, $preview_links) {
+  private function templateDescription($name, $email, $title, $id, $moderation_state, $bundle, $is_new, $updated_date, $notes, $preview_links) {
     return <<<EOT
-Hi Support,
-
 This page is ready for review.
 
-Editor information
+*Editor information*
 
 Editor name:   $name
 
 Editor email:   $email
 
-Department:   $department
-
-Page information
+*Page information*
 
 Page name:     $title
 
-CMS URL:         https://content.vic.gov.au/node/$id
+CMS URL:         https://content.vic.gov.au/node/$id/revisions
```

Comment:
is there a way to get the CMS URL automated way, so that if it is not content-vic and some other project, it can get that CMS URL as well?

Reply:
Fixed!